packages/metrics-app/src/utils/address.ts

```diff
@@ -0,0 +1,19 @@
+import { getCurrentCoordinates } from '../services/devicelocation'
+
+
+const BIGDATACLOUD_API_KEY = 'bdc_ec1c6c097de9484c8db9633444fb8cca'
+
+export async function getAddress(): Promise<string | null> {
+  const coordinates = await getCurrentCoordinates()
+  const url = `https://api.bigdatacloud.net/data/reverse-geocode?latitude=${coordinates?.latitude}&longitude=${coordinates?.longitude}&localityLanguage=en&key=${BIGDATACLOUD_API_KEY}`
+  const response = await fetch(url)
+
+  if (!response.ok) {
+    throw new Error('Failed to fetch address!')
```

Comment:
Let's just return `null` here, throwing an error would cause the entire `performMeasurementsFromQuery` function to fail. Since the area is an optional field on the report, it is acceptable if it is null.